protocol/directory/directory.go

```diff
@@ -6,14 +6,15 @@
 // lookups, and monitoring.
 // It does not yet support key changes.
 
-package protocol
+package directory
 
 import (
 	"bytes"
 
 	"github.com/coniks-sys/coniks-go/crypto/sign"
 	"github.com/coniks-sys/coniks-go/crypto/vrf"
 	"github.com/coniks-sys/coniks-go/merkletree"
+	p "github.com/coniks-sys/coniks-go/protocol"
```

Comment:
Nit pick: We should probably be consistent with how we import some other coniks-go packages. We always import `protocol` as `p` which I think is fine, but `merkletree` sometimes is imported as `m` (e.g. https://github.com/coniks-sys/coniks-go/blob/master/protocol/consistencychecks.go#L13), and other times (like here) not. I think I've seen this with `directory` as well (`dir` vs the full name).

Reply:
Ok, I am fixing this.